extensions/ql-vscode/src/view/method-modeling/MultipleModeledMethodsPanel.tsx

```diff
@@ -47,19 +54,72 @@ export const MultipleModeledMethodsPanel = ({
     setSelectedIndex((previousIndex) => previousIndex + 1);
   }, []);
 
+  const handleAddClick = useCallback(() => {
+    const newModeledMethod: ModeledMethod = {
+      type: "none",
+      input: "",
+      output: "",
+      kind: "",
+      provenance: "manual",
+      signature: method.signature,
+      packageName: method.packageName,
+      typeName: method.typeName,
+      methodName: method.methodName,
+      methodParameters: method.methodParameters,
+    };
+
+    const newModeledMethods = [...modeledMethods, newModeledMethod];
+
+    onChange(newModeledMethods);
+    setSelectedIndex(newModeledMethods.length - 1);
+  }, [onChange, modeledMethods, method]);
+  const handleRemoveClick = useCallback(() => {
```

Comment:
```suggestion

  const handleRemoveClick = useCallback(() => {
```
Just suggesting adding a newline so it's easier to read.